CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+Package Contribution Guidelines
+===============================
+
+If you are submitting a new Spack package, please observe the
+following standards:
+
+1. Code must be [PEP-8 compliant](http://spack.readthedocs.io/en/latest/contribution_guide.html?highlight=pep-8#flake8-tests)
+
```

Comment:
This list is specific for adding packages.  It was not meant to address contributions to documentation or core Spack.  Maybe we needs separate lists for that in CONTRIBUTING.md?